packages/backend/server/src/modules/workspaces/resolvers/blob.ts

```diff
@@ -70,29 +87,6 @@ export class WorkspaceBlobResolver {
     return { size };
   }
 
-  /**
-   * @deprecated mutation `setBlob` will check blob limit & quota usage
-   */
-  @Query(() => WorkspaceBlobSizes, {
-    deprecationReason: 'no more needed',
-  })
-  async checkBlobSize(
```

Comment:
we haven't implemented client version checking logic, better to keep deprecated API until that time, otherwise users will see the old clients complain error for no reason.